config/dune

```diff
@@ -0,0 +1,8 @@
+(executables
+ (names discover)
+ (libraries dune.configurator))
+
+(rule
+ (targets cclib.sexp)
+ (deps    discover.exe)
+ (action  (run ./discover.exe)))
```

Comment:
I prefer to have something like:

```
 (deps (:exe discover.exe))
 (action (run %{exe}))
```

But it's ok.

Reply:
I wasnt quite sure how that would interact with having multiple `discover.exe` in the same duniverse repo..